Bio/PDB/parse_pdb_header.py

```diff
@@ -77,7 +88,17 @@ def _format_date(pdb_date):
         "Nov",
         "Dec",
     ]
-    month = str(all_months.index(pdb_date[3:6]))
+    month_name = pdb_date[3:6]
```

Comment:
```python
    try:
        month = "%02d" % all_months.index(month_name)
    except ValueError:
        if not permissive:
            raise ValueError(f"Non-standard month supplied: {month_name}.") from None
        month = "00"
```

Reply:
That works - except we still need to raise a warning if it is permissive.

Reply:
You could, or decide in permissive mode just treat the month as zero. That's a judgement call - and right now your files are the only example of this I'm aware of.

Reply:
As per your comment 'where they break the specification it is reasonable to give a warning or error.' - I have put a warning in where they break the spec in permissive mode.